projects/plugins/jetpack/_inc/client/components/settings-card/index.jsx

```diff
@@ -86,6 +90,20 @@ export const SettingsCard = inprops => {
 		backupsEnabled = vpData?.data?.features?.backups ?? false,
 		scanEnabled = vpData?.data?.features?.security ?? false;
 
+	// Build a direct checkout URL to Business on WordPress.com when applicable
+	const isWpcom = typeof isWpcomPlatformSite === 'function' ? isWpcomPlatformSite() : false;
```

Comment:
Looks like this will be executed on each rerender, so maybe it's better to wrap it in a hook and also extracting the whole logic into a separate hook would be great.